spring-boot-project/spring-boot-parent/pom.xml

```diff
@@ -229,6 +229,25 @@
 				<artifactId>zt-zip</artifactId>
 				<version>1.7</version>
 			</dependency>
+			<dependency>
```

Comment:
Dependencies are (meant to be) ordered alphabetically. Can you please rework this?